homeassistant/components/daikin/config_flow.py

```diff
@@ -9,7 +9,7 @@
 from pydaikin.discovery import Discovery
 import voluptuous as vol
 
-from homeassistant import config_entries
+from homeassistant import config_entries, data_entry_flow
```

Comment:
dont think youll need this with other suggestion

Reply:
You are right.

Reply:
```suggestion
from homeassistant import config_entries
```